r2/r2/templates/modaction.xml

```diff
@@ -44,9 +44,9 @@
   if thing.details_text:
     title += ' (%s)' % thing.details_text
 
-  if isinstance(thing, Comment):
+  if  hasattr(thing, "target") and isinstance(thing.target, Comment):
```

Comment:
The below line, I *did* change `make_permalink_slow` to `make_permalink`, but it is not showing up. I assume you guys already did it in a commit that is not public, but, that was not all that needed to be done.